internal/driver/glfw/window_common.go

```diff
@@ -0,0 +1,36 @@
+package glfw
+
+const defaultTitle = "Fyne Application"
+
+func unknownKey(key string) bool {
+	if len(key) != 1 {
+		return true
+	}
+
+	keyRune := key[0]
+	if keyRune >= 'a' || keyRune <= 'z' {
+		return false
+	}
+
+	_, ok := keyKnownRuneMap[keyRune]
+	return !ok
+}
+
+// keyKnownRuneMap does not contain 'a' to 'z'. They are handled outside.
+var keyKnownRuneMap = map[byte]struct{}{
+	'\'': {},
+	',':  {},
+	'-':  {},
+	'.':  {},
+	'/':  {},
+	'*':  {},
+	'`':  {},
+
+	';': {},
+	'+': {},
+	'=': {},
+
+	'[':  {},
+	'\\': {},
+	']':  {},
+}
```

Comment:
This feels ugly to me. It might be worth it to just move the checks into the function but I am honestly not perfectly happy with that approach either.

There is some overlap in the ascii table that avoids having to check each character individually though.

Reply:
I removed it entirely. There are quite a few overlaps and it made sense to handle them in code instead of using a map.